parser-typechecker/src/Unison/Server/Backend.hs

```diff
@@ -444,15 +444,14 @@ searchBranchExact len names queries =
   in
     [ searchTypes q <> searchTerms q | q <- queries ]
 
-hqNameQuery'
+hqNameQuery
   :: Monad m
-  => Bool
-  -> Maybe Path
+  => Maybe Path
   -> Branch m
   -> Codebase m v Ann
   -> [HQ.HashQualified Name]
   -> m QueryResult
-hqNameQuery' doSuffixify relativeTo root codebase hqs = do
+hqNameQuery relativeTo root codebase hqs = do
```

Comment:
Removed this `doSuffixify` parameter, but @runarorama I don't totally understand what its purpose was - was it just an efficiency thing, to avoid doing suffixification in some cases? 

It doesn't seem harmful to do lookup by suffix.